src/Application/Console/Formatters/Console.php

```diff
@@ -541,7 +541,20 @@ private function getFileLinkFormatter(): FileLinkFormatter
         return $this->fileLinkFormatter;
     }
 
-    private function formatFileLine(Details $detail, string $directory): string
+    private function getCategoryColor(string $category): string
+    {
+        //black, red, green, yellow, blue, magenta, cyan, white, default
+        $categoryColor = [
+            'Code' => 'cyan',
+            'Complexity' => 'green',
+            'Architecture' => 'blue',
+            'Style' => 'yellow',
+            'Security' => 'red',
+        ];
+        return $categoryColor[$category] ?? '';
```

Comment:
Just to be sure, so if the category does not exist we return an empty string as the color. 
Does that actually work, will that result in no coloring or a crash?

Reply:
I think I left that in there to set the default color, I should have added a TODO 😅
Lets say we end up choosing blue for any unknown category 
```suggestion
        return $categoryColor[$category] ?? 'blue';
```

Reply:
Sounds perfect, let's do that.

Reply:
I had to commit it I think